src/scripts.js

```diff
@@ -135,8 +145,14 @@ const hydrationCardHandler = () => {
   if (event.target.classList.contains('user-ounces-submit')) {
     event.preventDefault();
     let input = document.querySelector('#input-ounces');
-    let hydrationObj = {userID: currentUser.id, date: todaysDate, numOunces: input.value};
-    currentUser.hydrationInfo.individualEntryRecords.unshift(hydrationObj);
+    let hydrationObj = {userID: currentUser.id, date: todaysDate, numOunces: Number(input.value)};
+    let matchedToday = currentUser.hydrationInfo.individualEntryRecords.find(hydroPoint => hydroPoint.date === hydrationObj.date)
+    if (matchedToday) {
+      matchedToday.numOunces = matchedToday.numOunces + hydrationObj.numOunces;
+    } else {
+      currentUser.hydrationInfo.individualEntryRecords.push(hydrationObj);
```

Comment:
Great work with this logic -- if there's already been an entry made for the day, then add data to the existing record. Otherwise make a new record. We should apply this to the other classes too.